kyaml/yaml/types.go

```diff
@@ -724,20 +724,29 @@ func (rn *RNode) UnmarshalJSON(b []byte) error {
 
 // ConvertJSONToYamlNode parses input json string and returns equivalent yaml node
 func ConvertJSONToYamlNode(jsonStr string) (*RNode, error) {
-	var body map[string]interface{}
-	err := json.Unmarshal([]byte(jsonStr), &body)
+	yml, err := ConvertJSONToYamlString(jsonStr)
 	if err != nil {
 		return nil, err
 	}
-	yml, err := yaml.Marshal(body)
+	node, err := Parse(yml)
 	if err != nil {
 		return nil, err
 	}
-	node, err := Parse(string(yml))
+	return node, nil
+}
+
+// ConvertJSONToYamlNode parses input json string and returns equivalent yaml string
+func ConvertJSONToYamlString(jsonStr string) (string, error) {
```

Comment:
nit: ConvertJSONToYAMLString

Reply:
Done

Reply:
The changes don't seem to be pushed.  Its still `ConvertJSONToYamlString`

Reply:
It did. But it doesn't reflect here. Please visit Files Changed tab.

Reply:
I am in the files changed tab and don't see it.

Reply:
Done.